cmake/ConfigureCompiler.cmake

```diff
@@ -351,6 +355,14 @@ else()
       -fno-builtin-realloc
       -fno-builtin-free)
   endif()
+  # SQLite won't build with -ffast-math
+  if (ICX)
```

Comment:
Should we disable those flags altogether for the entire compilation? Or should we disable them in SQLite only?